comid/example_test.go

```diff
@@ -588,3 +588,79 @@ func Example_decode_CBOR_4() {
 
 	// Output: OK
 }
+
+func Example_decode_CBOR_5() {
+	// Taken from https://github.com/ietf-corim-cddl/blob/main/examples/comid-2.diag
```

Comment:
we agreed this would go, at least until we add a new test vector into the ietf-corim-cddl repo that we can reference, right?